jetty-core/jetty-websocket/jetty-websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/util/MethodHolder.java

```diff
@@ -227,4 +236,37 @@ public Class<?> returnType()
             return _methodHandle.type().returnType();
         }
     }
+
+    class Wrapper implements MethodHolder
+    {
+        public MethodHolder _methodHolder;
+
+        public Wrapper(MethodHolder methodHolder)
+        {
+            _methodHolder = methodHolder;
+        }
+
+        public MethodHolder getWrapped()
+        {
+            return _methodHolder;
+        }
+
+        @Override
+        public Object invoke(Object... args) throws Throwable
+        {
+            return _methodHolder.invoke(args);
```

Comment:
```suggestion
            return getWrapped().invoke(args);
```

And also for the other methods.